File: src/CodeWars_Javascript/vowel_count.js
/*
https://www.codewars.com/kata/54ff3102c1bad923760001f3/
Given a String, return the number of vowels in the string
 */
console.log(getCount("Aaa"));

function getCount(str) {
    return str.split("").filter(q => q.toLowerCase() === 'a' || q === 'e' || q === 'i' || q === 'o'  || q === 'u' ).length;
}

File: src/StreamExamples/vowelCount.java
package StreamExamples;
//https://www.codewars.com/kata/54ff3102c1bad923760001f3

/*
Given a string, count the vowels.
 */
public class vowelCount {
    public static void main(String[] args) {
        System.out.println(getCount("abracadabra"));
    }
    public static long getCount(String str) {
        return  str.chars()
                .mapToObj(c -> (char) c)
                .filter(y -> y == 'a' || y == 'i' || y == 'e' || y == 'o' || y == 'u')
                .count();
    }

}
